### ISSUE TITLE
no proper .gitignore (r5_dockerize, with_git)

### ISSUE THREAD

**Issue by themisa** [clustervision/trinity#418]
In r5_dockerize (and with_git)

**Version:** r5_dockerize 8bfa843

there is not proper .gitignore file (missing the .) and creating the repo takes a long amount of time

git status:
```
[root@controller(unknownbin boot cluster dev etc gitignore home install lib lib64 media mnt opt proc root run sbin srv sys tftpboot tmp trinity usr var xcatpost) /]# git status
# On branch master
#
# Initial commit
#
# Untracked files:
#   (use "git add <file>..." to include in what will be committed)
#
#       .autorelabel
#       .readahead
#       bin
#       boot/
#       cluster/
#       dev/
#       etc/
#       gitignore
#       home/
#       install/
#       lib
#       lib64
#       opt/
#       proc/
#       root/
#       run/
#       sbin
#       sys/
#       tftpboot/
#       tmp/
#       trinity/
#       usr/
#       var/
#       xcatpost/
nothing added to commit but untracked files present (use "git add" to track)
[root@controller(unknownbin boot cluster dev etc gitignore home install lib lib64 media mnt opt proc root run sbin srv sys tftpboot tmp trinity usr var xcat
```

**Comment by hansthen** [comment]
This is fixed in the with_git branch 83c2ef04069c4ae34b892934dd42adf71b3a0324

**Comment by themisa** [comment]
Fixed. 
Tested in:

**Version:** r5_dockerize 1e9bf1e
**Environment:** dmaster

```
[root@controller(dmaster*) ~]# git status
# On branch master
# Changes not staged for commit:
#   (use "git add <file>..." to update what will be committed)
#   (use "git checkout -- <file>..." to discard changes in working directory)
#
#       modified:   ../etc/resolv.conf
#
no changes added to commit (use "git add" and/or "git commit -a")
```
 
```
[root@controller(dmaster*) /]# more .gitignore 
/*
*.rpm
!etc/
/etc/xcat/*.sqlite
!install/custom/
!install/postscripts/
```
